rgain3/lib/__init__.py

```diff
@@ -37,36 +38,10 @@ class GainData:
      - ``ref_level``: the used reference level (in dB)
     """
 
-    def __init__(self,
-                 gain,
-                 peak=1.0,
-                 ref_level=89,
-                 gain_type=GainType.TP_UNDEFINED):
-        self.gain = gain
-        self.peak = peak
-        self.ref_level = ref_level
-        self.gain_type = gain_type
-
-    def __str__(self):
-        return "gain={.2f} dB; peak={.8f}; reference-level={} dB".format(
```

Comment:
The syntax error was caused by the float fomatting, which misses the colon and should be `{:2f}`.